docs/editors-guide/merging-and-obsoleting.md

```diff
@@ -46,34 +46,43 @@ See [GitHub Discussion](https://github.com/monarch-initiative/mondo/discussions/
     1. Mondo ID for the term that will replace the obsoleted terms
     1. Label for the term that will replace the obsoleted terms
     
-### Workflow video explanation
-See [here](https://drive.google.com/file/d/1pnf0BT1qJ9wxXKvykkWhjwvFLceI2U-2/view?usp=sharing)
-    
-#### If new obsoletion reasons are needed
+    #### Video explanation of Workflow
+
+    See video explanation <a href="https://drive.google.com/file/d/1pnf0BT1qJ9wxXKvykkWhjwvFLceI2U-2/view?usp=sharing" target="_blank">here.</a>
```

Comment:
I think you can embed the video by doing this- but maybe it the video has to be in YouTube?

<iframe width="560" height="315" src="https://www.youtube.com/embed/WNld_e0XdQ8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

this example is from: 
https://oboacademy.github.io/obook/howto/open-science-engineer/

Reply:
I see now that you know about this since you did it below

Reply:
The others were already embedded and YouTube videos. Using `iframe` does not work with the gDrive video.